contributing.md

```diff
@@ -134,6 +195,16 @@ Entropy source is a function that gets a piece of data about the browser a.k.a.
 Entropy sources are located in the [src/sources](src/sources) directory.
 Entropy component must be a simple JS value that can be JSON-encoded.
```

Comment:
I think we should be consistent with calling it wither "Entropy sources" or "entropy components" and avoid using multiple alternatives, e.g. "Fingerprint component". 
Also: "Entropy sources must be simple JavaScript values that can be encoded into JSON".

Reply:
IIRC you used terms "entropy source" (a function) and "fingerprint component" (the return value) in v2. We keep this terminology, but we've changed "fingerprint component" to "entropy component" because we use `visitorId` instead of `fingerprint` in v3 to keep API compatibility with the Pro version. So I'd like to keep both "entropy source" and "entropy component" and remove "fingerprint component".

I don't mind using a single term for both function and its return value (we use "signal" internally), but I think it should be a separate PR because it's an extensive change.